packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.controller.js

```diff
@@ -20,6 +20,11 @@ export default class TechnicalDetailsController {
     );
   }
 
+  ramUpgradable() {
+    const ram = get(this.technicalDetails, 'memory');
```

Comment:
some nitpicking :
since you are not reusing the "ram" object, i guess you can simply do :
```js
const upgradableLength = get(this.technicalDetails, 'memory.upgradable.length', 0);
return this. upgradeWithTicketAvailable && upgradableLength > 0;
```

also for function returning booleans a better name would be ```isRamUpgradable()```

Reply:
@Heiiji what's about function renaming as @frenautvh has mentioned?